src/tests/ftest/control/daos_system_query.yaml

```diff
@@ -0,0 +1,27 @@
+hosts:
+  test_servers: 1
+timeout: 80
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      targets: 8
```

Comment:
Unless this test specifically needs to specify `targets`, I'd recommend removing this. The default in ftest is 8
```suggestion
```

Reply:
Removed.